BruteForceTest#testResetPassword failure

### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

ci

### Describe the bug

This test is currently failing when running with Chrome or Firefox (default HtmlUnitDriver is passing):

```
[ERROR] org.keycloak.testsuite.forms.BruteForceTest.testResetPassword  Time elapsed: 17.891 s  <<< FAILURE!
java.lang.AssertionError
        at org.junit.Assert.fail(Assert.java:87)
        at org.junit.Assert.assertTrue(Assert.java:42)
        at org.junit.Assert.assertFalse(Assert.java:65)
        at org.junit.Assert.assertFalse(Assert.java:75)
        at org.keycloak.testsuite.forms.BruteForceTest.testResetPassword(BruteForceTest.java:589)
```


### Version

latest

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

`mvn test -Pauth-server-quarkus -f testsuite/integration-arquillian/tests/base/pom.xml -Dtest=BruteForceTest#testResetPassword -Dbrowser=firefox`

or

`mvn test -Pauth-server-quarkus -f testsuite/integration-arquillian/tests/base/pom.xml -Dtest=BruteForceTest#testResetPassword -Dbrowser=chrome -Dwebdriver.chrome.driver=/usr/local/bin/chromedriver`

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/AbstractPage.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/AbstractPage.java
@@ -23,7 +23,6 @@ import org.keycloak.common.util.KeycloakUriBuilder;
 import org.keycloak.testsuite.arquillian.SuiteContext;
 import org.keycloak.testsuite.util.DroneUtils;
 import org.keycloak.testsuite.util.OAuthClient;
-import org.keycloak.testsuite.util.WaitUtils;
 import org.openqa.selenium.WebDriver;
 
 import java.net.URI;
@@ -44,7 +43,6 @@ public abstract class AbstractPage {
     protected OAuthClient oauth;
 
     public void assertCurrent() {
-        WaitUtils.waitForPageToLoad();
         String name = getClass().getSimpleName();
         Assert.assertTrue("Expected " + name + " but was " + DroneUtils.getCurrentDriver().getTitle() + " (" + DroneUtils.getCurrentDriver().getCurrentUrl() + ")",
                 isCurrent());

